### ISSUE TITLE
python setup.py install 报错

### ISSUE THREAD

**Issue by huichengxiao** [aksnzhy/xlearn#261]
首先我在windows文件夹用 Visual Studio2017编译成功了，然后我进入xlearn\python-package, 打开命令行， 输入python setup.py install,结果报错如下：
Traceback (most recent call last):
  File "setup.py", line 29, in <module>
    LIB_PATH = [os.path.relpath(libfile, CURRENT_DIR) for libfile in libpa
nd_lib_path']()]
  File "xlearn/libpath.py", line 63, in find_lib_path
    'Cannot find xlearn Library in the candidate path'
XLearnLibraryNotFound: Cannot find xlearn Library in the candidate path
我看也有其他用户遇到这个问题，请问怎么解决？

**Comment by aksnzhy** [comment]
@etveritas Could you please address this issue? Thanks.

**Comment by etveritas** [comment]
@huichengxiao 你好，我想是编译前没有将编译选项改成``Release``模式，项目默认加载是``Debug``模式，你可以在VS的导航栏上将其改成``Release``，然后再对``xlearn_api``这个项目进行编译。

**Comment by mrlonely001** [comment]
version 0.4.4 还未支持pip 安装？

**Comment by etveritas** [comment]
@mrlonely001 在[release](https://github.com/aksnzhy/xlearn/releases)里我们提供了编译好的二进制库，可以从那里下载安装。

**Comment by huichengxiao** [comment]
谢谢，编译安装成功了

**Comment by wmmxk** [comment]
> @etveritas Could you please address this issue? Thanks.
I also run into this when installing from source. I could compile the source code by running make. When I run ./install-python.sh, I run into this error: XLearnLibraryNotFound: Cannot find xlearn Library in the candidate path. 

Then I looked into the setup.py file and found the path for the complied libxlearn_api.so was not detected correctly in the  [libpath.py](https://github.com/aksnzhy/xlearn/blob/9b6754284e86fc43231b52264c6f33bb83b144c3/python-package/xlearn/libpath.py#L36). The compiled file was in the "../../build/lib/" folder. So I changed "../../lib" to "../../build/lib". Now it works.
